components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/JITProvisioningPostAuthenticationHandler.java

```diff
@@ -486,7 +486,9 @@ private void handleAccountLockLoginFailure(String retryPage, AuthenticationConte
             boolean showAuthFailureReason = true;
             retryPage = FrameworkUtils.appendQueryParamsStringToUrl(retryPage,
                     "sp=" + context.getServiceProviderName());
-                                                                    
+            retryPage = FrameworkUtils.appendQueryParamsStringToUrl(retryPage,
+                    String.format("%s=", FrameworkConstants.REQUEST_PARAM_AUTH_FLOW_ID)
```

Comment:
Any reason for using formatting and concatenation both here?
If we are using String.format why not use "%s=%s"